app/src/main/res/layout/activity_settings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ui.SettingsActivity">
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
You should a basic View as a container, then use a settings fragment inside an all-but-empty SettingsActivity.

See here: https://developer.android.com/guide/topics/ui/settings/

Reply:
![image](https://user-images.githubusercontent.com/3866197/52679851-6ad17f80-2f04-11e9-9ad1-646dec365768.png)

Alas, I haven't been able to figure out what's going on with settings fragments.

Reply:
This will get done later; however, it would be nice to have something linked to the app bar for now.
Can we go for another PR on this?